Tests/CryptorRSATests/CryptorRSATests.swift

```diff
@@ -445,11 +476,12 @@ class CryptorRSATests: XCTestCase {
             -----END PUBLIC KEY-----
             """
         
-        let tokenPublicKey = try? CryptorRSA.createPublicKey(withPEM: certificatePEM)
+        let tokenPublicKey = try CryptorRSA.createPublicKey(withPEM: certificatePEM)
```

Comment:
Better to `guard let` this, `XCTFail(); return` in the `else`, and remove the `XCTAssertNotNil()` below.